plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineFacet.java

```diff
@@ -0,0 +1,13 @@
+package com.google.cloud.tools.eclipse.appengine.facets;
+
+public class AppEngineFacet {
+
+  public static final String APP_ENGINE_FACET_ID = "com.google.cloud.tools.eclipse.appengine.facet";
```

Comment:
perhaps just ID so it's AppEngineFacet.ID instead of AppEngineFacet.APP_ENGINE_FACET_ID